jni/lib/src/jni_object.dart

```diff
@@ -162,14 +97,14 @@ int _getCallType(int? callType, int defaultType, Set<int> allowed) {
   throw InvalidCallTypeException(callType, allowed);
 }
 
-T _callOrGet<T>(int? callType, Function(int) f) {
+T _callOrGet<T>(int? callType, JniResult Function(int) f) {
```

Comment:
Maybe give `f` a better name, then the calls with `f(...)` are a bit easier to see further down below.